OpenOversight/migrations/env.py

```diff
@@ -62,7 +50,7 @@ def run_migrations_online():
     # this callback is used to prevent an auto-migration from being generated
     # when there are no changes to the schema
     # reference: http://alembic.zzzcomputing.com/en/latest/cookbook.html
-    def process_revision_directives(context, revision, directives):
```

Comment:
The first two parameters are not being used and are masking outside variables, so I removed them.